using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -705,7 +860,41 @@ If you have an [Experience Cloud ID](https://app.gitbook.com/@aep-sdks/s/docs/us
 {% endhint %}
 
 {% tabs %}
-{% tab title="iOS" %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+### getTrackingIdentifier
+
+Retrieves the Analytics tracking identifier.
+
+#### Syntax
+
+```swift
+static func getTrackingIdentifier(completion: @escaping (String?, Error?) -> Void)
+```
+
+#### Example
+
+**Swift**
+
+```swift
+Analytics.getTrackingIdentifier { (trackingId, error) in
+   // check the trackingIdentifier value  
+}
+```
+
+**Objective-C**
+
+```objectivec
+AEPMobileAnalytics getTrackingIdentifier:^(NSString * _Nullable trackingIdentifier, NSError * _Nullable error) {
+   // check the trackingIdentifier value  
+}];
+```
+
+{% endtab %}
```

Comment:
Same comment here regarding the duplicated section.

If we only have one API in AEP, one thing we can do is to document it as part of the getTrackingIdentifier section and just provide a link to it in the getTrackingIdentifierWithCompletionHandler section.